### ISSUE TITLE
The Topcoder time is confusing when its shown as UTC.

### ISSUE THREAD

**Issue by neeths81** [topcoder-archive/beta-testing-dashboard#40]
![image](https://user-images.githubusercontent.com/68718750/133471184-23f940ed-6cad-4994-afb6-8ce0e4302ae7.png)
It will be good if the time is displayed based on the location automatically instead of showing as UTC

**Comment by jmpld40** [comment]
UTC-4 is the most globally friendly time. How else can we add time to help you? Maybe link to: see what time it is in your area?

**Comment by neeths81** [comment]
Yes, may be if there is an option to see my local time instead of UTC will be very straight forward to understand.
